modules/basics/src/main/java/com/opengamma/strata/basics/currency/Money.java

```diff
@@ -142,6 +152,98 @@ public BigDecimal getAmount() {
     return amount;
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Returns a copy of this {@code Money} with the specified amount added.
+   * <p>
+   * This adds the specified amount to this monetary amount, returning a new object.
+   * <p>
+   * This instance is immutable and unaffected by this method.
+   * 
+   * @param amountToAdd  the amount to add, in the same currency
+   * @return an amount based on this with the specified amount added
+   * @throws IllegalArgumentException if the currencies are not equal
+   */
+  public Money plus(Money amountToAdd) {
+    ArgChecker.notNull(amountToAdd, "amountToAdd");
+    ArgChecker.isTrue(amountToAdd.getCurrency().equals(currency), "Unable to add amounts in different currencies");
+    return new Money(currency, amount.add(amountToAdd.amount));
+  }
+
+  /**
+   * Returns a copy of this {@code Money} with the specified amount subtracted.
+   * <p>
+   * This subtracts the specified amount to this monetary amount, returning a new object.
+   * <p>
+   * This instance is immutable and unaffected by this method.
+   * 
+   * @param amountToSubtract  the amount to subtract, in the same currency
+   * @return an amount based on this with the specified amount subtracted
+   * @throws IllegalArgumentException if the currencies are not equal
+   */
+  public Money minus(Money amountToSubtract) {
+    ArgChecker.notNull(amountToSubtract, "amountToSubtract");
+    ArgChecker.isTrue(amountToSubtract.getCurrency().equals(currency), "Unable to subtract amounts in different currencies");
+    return new Money(currency, amount.subtract(amountToSubtract.amount));
+  }
+
+  //-------------------------------------------------------------------------
+  /**
+   * Returns a copy of this {@code Money} with the amount multiplied.
+   * <p>
+   * This takes this amount and multiplies it by the specified value.
+   * <p>
+   * This instance is immutable and unaffected by this method.
+   * 
+   * @param valueToMultiplyBy  the scalar amount to multiply by
+   * @return an amount based on this with the amount multiplied
+   */
+  public Money multipliedBy(long valueToMultiplyBy) {
```

Comment:
Is it worth having `public Money multipliedBy(BigDecimal valueToMultiplyBy)` too?

Reply:
The problem is rounding. Anything other than an integer needs rounding.